Healthy/Classes/Extensions/UIFont+Style.swift

```diff
@@ -0,0 +1,54 @@
+import UIKit
+
+// MARK: add appFont func
+
+extension UIFont {
+
+    private enum AppWeight {
+        case regular
+        case bold
+    }
+
+    private static func appFont(ofSize size: CGFloat, weight: AppWeight) -> UIFont {
+
+        let fontName: String = {
+            switch weight {
+            case .regular:
+                return "Poppins-Regular"
+            case .bold:
+                return "Poppins-Bold"
+            }
+        }()
+
+        if let font = UIFont(name: fontName, size: size) {
+            return font
+        } else {
+            return UIFont.systemFont(ofSize: size, weight: weight == .regular ? .regular : .bold)
+        }
+
+    }
+}
+
+// MARK: - Bold Fonts
+
+extension UIFont {
+    static let tittleTextBold   = appFont(ofSize: 50, weight: .bold)
+    static let headerTextBold   = appFont(ofSize: 30, weight: .bold)
+    static let largeTextBold    = appFont(ofSize: 20, weight: .bold)
+    static let mediumTextBold   = appFont(ofSize: 18, weight: .bold)
+    static let normalTextBold   = appFont(ofSize: 16, weight: .bold)
+    static let smallTextBold    = appFont(ofSize: 14, weight: .bold)
+    static let smallerTextBold  = appFont(ofSize: 11, weight: .bold)
+}
+
+// MARK: - Regular Fonts
+
+extension UIFont {
+    static let tittleTextRegular    = appFont(ofSize: 50, weight: .regular)
+    static let headerTextRegular    = appFont(ofSize: 30, weight: .regular)
+    static let largeTextRegular     = appFont(ofSize: 20, weight: .regular)
+    static let mediumTextRegular    = appFont(ofSize: 18, weight: .regular)
+    static let normalTextRegular    = appFont(ofSize: 16, weight: .regular)
+    static let smallTextRegular     = appFont(ofSize: 14, weight: .regular)
+    static let smallerTextRegular   = appFont(ofSize: 11, weight: .regular)
+}
```

Comment:
Let's not store the fonts in memory, the system do this for us. Instead, please use computed properties.

Reply:
Remove **"Text"** keyword, it's redundant.